modules/mono/glue/GodotSharp/GodotPlugins/Main.cs

```diff
@@ -123,7 +127,14 @@ internal void Unload()
             }
             catch (Exception e)
             {
-                Console.Error.WriteLine(e);
+                if (isNativeFuncionsReady)
+                {
+                    ExceptionUtils.LogException(e);
+                }
+                else
+                {
+                    // TODO: Find a way for handle exception messaging before NativeFuncs is initialized
```

Comment:
At the very least we should print it to the console with `Console.Error.WriteLine` like before.